ko-search/movies-evan-likes/src/App/components/helpers/testdata.js

```diff
@@ -0,0 +1,32 @@
+export default () => [
+  {
+    title: 'Arrival',
+    year: '2016',
+    evanSays:
+      "Loved this movie! Cerebral, engaging science fiction at its best. Do yourself a favor and don't watch any trailers or read anything about it before seeing it. You won't be disappointed!",
+  },
+  {
+    title: 'Coherence',
+    year: '2013',
+    evanSays:
+      'Another fantastic science fiction, shot on an insanely low budget. Fun fact: there was actually no script for this movie. The director gave each actor/actress note cards the day-of filming with character motivations and suspicions, and the actors improvised the actual dialogue and acting. So, so, SO GOOD!',
+  },
+  {
+    title: 'Interstellar',
+    year: '2014',
+    evanSays:
+      'I know this movie got a lot of flak when it came out, with many disliking it passionately. I first saw it in IMAX and I was really emotionally moved by it\'s poignancy. I think the film posits the idea that the universe is so incomprehensibly huge that the only way to make sense of our place in it is with the human relationships we have with each other. The whole "love will guide me" trope is cliché, agreed, but that isn\'t going to stop me from really enjoying this moive for the spectacle it offers.',
+  },
+  {
+    title: 'Apollo 13',
+    year: '1995',
+    evanSays:
+      "One of my favorite space movies of all time. Makes me both incredibly proud as a member of the human race and incredibly frustrated we haven't done anything in space remotely close to what we did in the 60's & 70's since. NASA is awesome, and this film really feels great to watch. Not to mention it has an excellent musical score.",
+  },
+  {
+    title: 'Moon',
+    year: '2009',
+    evanSays:
+      'Basically a one-man show with Sam Rockwell, this is a fantastic movie and directorial debut by Duncan Jones. You\'ll enjoy it most if you go in not knowing anything. One of the things I love most about this film is how it plays with your understanding of "the twist": this movie is smart.',
+  },
+];
```

Comment:
Just thinking that you could've required the JSON file directly, then just used the first `n` records

```js
const data = require('./path/to/data.json')
const testData = data.slice(0, 3); // or whatever
```

It's a minor thing but would've saved making the duplicates. ;)